admin/partials/jtrt-adminClass-shortcode.php

```diff
@@ -104,7 +104,7 @@ function jtrt_shortcode_table( $atts ){
                             if($myTableResponsiveStyle == "stack"){
                                 $html .= "<td><span class='stackedheadtitlejt' style='font-weight:bold;'>". $table_data[0][$cellindx] .":</span><br>" .preg_replace("/[\n\r]/","<br>",$cell). "</td>";
```

Comment:
We will need the do_shortcode function here as well just in case they use other styles. Also add the same do shortcode to the header of the tables as well, it should be a bit above in this same file <th> .. etc